shared/src/main/scala/tastyquery/reader/TreeUnpickler.scala

```diff
@@ -588,9 +587,8 @@ class TreeUnpickler(protected val reader: TastyReader, nameAtRef: NameTable) {
       reader.readByte()
       val name = readName
       val typ = readType
-      // TODO: use type
       // TODO: this might be more complicated than Ident
-      Ident(name)
+      TermRefTree(name, typ)
```

Comment:
this could be tested on file `test-sources/src/main/scala/simple_trees/TypeApply.scala` (`Seq` selection is TERMREF)